recipes/xtl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xtl" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "97cb01feb042ee59727ea1dcf502ed300c3533f009a6e606094b2811c66ee1f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xtl/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xtl  # [unix]
```

Comment:
Not sure this test for the directory does anything given that the one below exists, no?

Reply:
Indeed this test could be removed.